### ISSUE TITLE
Request: Make toolbar buttons accessible by adding discernible text

### ISSUE THREAD

**Issue by galvarez421** [nhn/tui.editor#235]
## Version
1.1.1

## Test Environment
Windows 7 Enterprise, Google Chrome

## Current Behavior
The toolbar buttons do not have discernible text, which is an accessibility issue, per https://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-rsv. See https://dequeuniversity.com/rules/axe/3.0/button-name and https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html for summaries of the guideline and options for satisfying the guideline.

The issue can be observed on https://nhnent.github.io/tui.editor/api/latest/tutorial-example01-basic.html. If you inspect one of the toolbar buttons with developer tools, you can see that there is no text within the button, no aria-label attribute, etc.

## Expected Behavior
The buttons should have discernible text to satisfy WCAG accessibility guidelines.
